docs/variants/novacustom_ns5x_adl/releases.md

```diff
@@ -13,6 +13,80 @@ For details about our release process please read
 
 </center>
 
+## v1.7.2 - 2023-12-22
+
+Test results for this release can be found
+[here](https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit#gid=799846462).
```

Comment:
@mkopec please fix spreadsheet link to: https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit#gid=271422464